rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1484,6 +1485,10 @@ protected void doCALL(){
 
         PrecompiledContracts.PrecompiledContract precompiledContract = precompiledContracts.getContractForAddress(activations, codeAddress);
 
+        // For gas Exactimation, mark this program execution if it requires
+        // at least one call with value
+        // todo(fedejinich) remove this comment before merging it into master
```

Comment:
Mhmm, probably this comment is outdated?

Reply:
it should be removed (on it)

Reply:
https://github.com/rsksmart/rskj/pull/1548/commits/dec990f8a6d945e3ce958b757b726c56b416fb8f